### ISSUE TITLE
Simplify Induced charge when dragging or jumping right

### ISSUE THREAD

**Issue by terracoda** [phetsims/balloons-and-static-electricity#389]
Based on testing in [#385](https://github.com/phetsims/balloons-and-static-electricity/issues/385) induced charge dragging descriptions can sound awkward.

After testing thoroughly dragging and jumping alerts in issue #385, I realize that we can often get the awkward order of the generic description “move away”, and then the begin description “begin to move away” when movement continues right (towards the wall). I think removing the “begin to move away” parameter all together and using the "generic" induced charge description  would fix this awkwardness.

**Logic**
1. First visible movement of charges in wall use [[move away]] (i.e., [+ Induced Charge Dragging (generic).])
2. Horizontal progression right, use [[move away a little more]]
3. Vertical dragging with visible induced charge just use [+ Induced Charge Dragging (generic).]

I've updated the design documents and the examples (https://docs.google.com/document/d/1WNVq5_-Hc9LSdeZVSn2edlwYwpHtGNT1ThFD9NXpZ5Q/edit#heading=h.30z1g3pdaiq4)

This design change should simplify what was fixed in closed issue #345.

This sim is in Dev Testing, so tagging phetsims/QA#112.

**Comment by terracoda** [comment]
@jessegreenberg, because this is design change, I have added design:a11y and dev:a11y labels. If you think this change will cause an implementation problem, please remove the dev:a11y label and assign it back to me, and I will investigate further.

As far as I can tell, I think this change simplifies the design in a good way.

**Comment by jessegreenberg** [comment]
Change made in the above commit, initial  induced charge change uses generic description.

**Comment by jessegreenberg** [comment]
@terracoda due to build process changes I am not able to deploy a dev version. Can you please verify that this is fixed on phettest?

**Comment by terracoda** [comment]
@jessegreenberg, I think this sounds good! As before, when I use the jump keys I get the generic description, and then on the next right-ward move I get the generic description again, and then on the second step right after a jump I get the "move away a little more" description. Hearing 2 instances of the generic description sounds way less awkward than the generic "move away", then the "begin to move away", and then the continuous "move away a little more" as rightward progression continues.

I did notice (3 times only) that the first vertical movement after a jump, I got the "begin to return" descriptions. This seemed to be random, but it happened twice when I jumped left to center and then moved up. I think I will comment on this more in issue #385 as I try to reproduce the issue. I could no longer reproduce this issue with horizontal movement. 

Since you have implemented the change as suggested and I verified it sounds good. I think we can close this issue.
